tests/unit_tests/test_internal_api.py

```diff
@@ -99,6 +99,218 @@ def test_internal_api_with_no_write_global_config_dir(tmp_path):
         os.chmod(tmp_path, 0o777)  # Allow the test runner to clean up.
 
 
+def test_fetch_orgs_and_org_entities_from_entity(mock_server):
+    """Test fetching organization entities from an entity."""
+    api = wandb.Api()
```

Comment:
question: I couldn't find the `mock_server` fixture (may have missed it), so was wondering -- where is this defined and/or configured?

Reply:
ah I've had issues with api fixture before. I keep getting
wandb.errors.errors.UsageError: api_key not configured (no-tty). call wandb.login(key=[your_api_key])
/wandb/wandb/sdk/wandb_login.py:281: UsageError

Reply:
Ah my mistake, I just realized this was in `unit_tests` (instead of `system_tests`).  Not 100% sure if that's why, but it would make sense if the `api` fixture doesn't work here without extra mocking/patching